.github/workflows/push_build.yml

```diff
@@ -0,0 +1,40 @@
+name: Push build to docker hub
+on: push
+
+env:
+  IMAGE_NAME: ajitshrivastav/pastebin
+
+jobs:
+  publish_to_docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Env
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}  
```

Comment:
Security hardening check: Use of Github secret in workflow - https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow